### PR TITLE
Rename blockMany/blockList in version test

### PR DESCRIPTION
**Pull request by queengooborg**
This PR renames `blockMany` and `blockList` in the version tests to something that's more self-explanatory.

### CODE REVIEW

File: test/linter/test-versions.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ for (const browser of Object.keys(browsers)) {
 }
 
 /** @type {string[]} */
-const blockMany = [
+const realValuesTargetBrowsers = [
```
Comment (caugner) [inline test/linter/test-versions.js:53]: Is this essentially `browsersRequiringNumericalVersions`? Or what does the "real" refer to?

File: test/linter/test-versions.js [hunk 2]
```diff
@@ -66,15 +66,15 @@ const blockMany = [
 ];
 
 /** @type {object.<string, string[]>} */
-const blockList = {
-  api: blockMany,
-  css: blockMany,
+const realValuesRequired = {
```
Comment (caugner) [inline test/linter/test-versions.js:69]: Ditto, is this bascially `browsersRequiringNumericalVersionsBySection` (assuming that "api", "css" etc. are sections, not sure if this is the right term)?

File: test/linter/test-versions.js (hunk 1 above)
Comment (caugner) [reply test/linter/test-versions.js:53]: Related question: Why do onlyu some browsers require "real" values?

Comment (queengooborg) [reply test/linter/test-versions.js:53]: That's correct -- we've used the terms "real" and "nonreal" to describe exact version numbers and boolean/null values respectively throughout BCD!

**Pull request merged**